hyperemitter.js

```diff
@@ -26,7 +26,7 @@ function HyperEmitter (db, schema, opts) {
     return new HyperEmitter(db, schema, opts)
   }
 
-  this.messages = protobuf(schema)
+  this.codecs = protobuf(schema)
```

Comment:
On second thought, I don't want to bump the major now. So, if the argument
is a string, keep the old behavior, but add a deprecation warnig. And add
an alias from 'codecs' to messages. You can use an ES5 object property for
that, and add a deprecation warning too.
Il giorno dom 24 mag 2015 alle 12:50 Dean McDonnell <
notifications@github.com> ha scritto:

> In hyperemitter.js
> <https://github.com/mcollina/hyperemitter/pull/15#discussion_r30951723>:
>
> > @@ -26,7 +26,7 @@ function HyperEmitter (db, schema, opts) {
> >      return new HyperEmitter(db, schema, opts)
> >    }
> >
> > -  this.messages = protobuf(schema)
> > +  this.codecs = protobuf(schema)
>
> I think this makes the most sense. On it now
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mcollina/hyperemitter/pull/15/files#r30951723>.
>